codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -185,7 +195,14 @@ public final T build(Http2Connection connection) {
                 reader = new Http2InboundFrameLogger(reader, frameLogger);
                 writer = new Http2OutboundFrameLogger(writer, frameLogger);
             }
+
+            // Configure the flow controllers.
+            connection.local().flowController(new DefaultHttp2LocalFlowController(connection, writer,
+                    DEFAULT_WINDOW_UPDATE_RATIO, autoRefillConnectionWindow));
+            connection.remote().flowController(new DefaultHttp2RemoteFlowController(connection));
```

Comment:
I had considered that, but they need the `frameWriter` to function properly.  Not sure how to allow them to specify a flow controller in that case.  Any thoughts?